apps/hpc-ftsadmin/src/libs/dayjs.ts

```diff
@@ -18,4 +18,33 @@ import 'dayjs/locale/zh';
 dayjs.extend(localizedFormat);
 dayjs.extend(relativeTime);
 
+export const FTS_DEFAULT_FORMAT = 'DD/MM/YYYY';
+
+// Extend the Dayjs class with the new format function
+dayjs.extend((_, DayjsClass) => {
+  const oldFormat = DayjsClass.prototype.format;
+
+  DayjsClass.prototype.format = function (formatString?: string) {
+    return oldFormat.bind(this)(formatString ?? FTS_DEFAULT_FORMAT);
+  };
+});
+
+declare module 'dayjs' {
+  interface Dayjs {
+    /**
+     * * This is a modified version of Dayjs format() function *
```

Comment:
This is a bad docstring, as it puts this text under a bullet point. Asterisk needs to be next to text, single for italics and double for bold
```suggestion
     * *This is a modified version of Dayjs format() function*
```